prowler/providers/aws/services/elasticache/elasticache_redis_cluster_below_v6_auth_enabled/elasticache_redis_cluster_below_v6_auth_enabled.py

```diff
@@ -0,0 +1,27 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.elasticache.elasticache_client import (
+    elasticache_client,
+)
+
+
+class elasticache_redis_cluster_below_v6_auth_enabled(Check):
+    def execute(self):
+        findings = []
+        for cluster in elasticache_client.clusters.values():
```

Comment:
Please, use the `replication_groups` since we have to check it at that level and update the metadata.